oauthenticator/github.py

```diff
@@ -54,6 +54,13 @@ def _github_client_secret_changed(self, name, old, new):
     client_secret_env = 'GITHUB_CLIENT_SECRET'
     login_handler = GitHubLoginHandler
     
+    github_to_nix_usernames_map = Dict(config=True, 
```

Comment:
I think this should just be called github_user_name_map, since these are not necessarily Local Unix Users - can be anything depending on the particular spawner being used.